pkgs/misc/lguf-brightness/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, libusb, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "lguf-brightness";
```

Comment:
This should be pname instead of name, if it were to be name it would need to contain the version of the package in the form of packagename-version